composer.json

```
@@ -15,7 +15,7 @@
         "symfony/css-selector": "^5.3|^6.0",
         "symfony/dom-crawler": "^6.0.3",
         "symfony/process": "^5.3|^6.0",
-        "illuminate/collections": "^9.1",
+        "illuminate/collections": "^8.23.1",
```

Mind updating this to a range? Then we can get this merged.

```suggestion
        "illuminate/collections": "^10.0|^9.1|^8.23.1",
```